src/QA/QA.Web/Views/Dashboard/Index.cshtml

```diff
@@ -1,59 +1,97 @@
+@using Esfa.Recruit.Vacancies.Client.Domain.Extensions
 @model DashboardViewModel
 <div class="grid-row">
     <div class="column-full">
         <h1 class="heading-xlarge no-bottom-margin">QA vacancies</h1>
         <hr>
     </div>
 </div>
-
+<form asp-route="@RouteNames.Dashboard_Index_Post" method="post">
+    <div class="grid-row">
+        <input asp-for="SearchTerm" placeholder="Search vacancies" />
+        <button type="submit">Search</button>
+    </div>
+</form>
 <div class="grid-row">
     <div class="column-full">
         <h2 class="heading-medium no-bottom-margin">Statistics</h2>
         <table>
             <thead>
-            <tr>
-                <th>Total vacancies</th>
-                <th>Submitted +24 hours</th>
-                <th>Resubmitted vacancies</th>
-            </tr>
+                <tr>
+                    <th>Total vacancies</th>
+                    <th>Submitted +24 hours</th>
+                    <th>Resubmitted vacancies</th>
+                </tr>
             </thead>
             <tbody>
-            <tr>
-                <td>@Model.TotalVacanciesForReview</td>
-                <td>@Model.TotalVacanciesBrokenSla</td>
-                <td>@Model.TotalVacanciesResubmitted</td>
-            </tr>
+                <tr>
+                    <td>@Model.TotalVacanciesForReview</td>
+                    <td>@Model.TotalVacanciesBrokenSla</td>
+                    <td>@Model.TotalVacanciesResubmitted</td>
+                </tr>
             </tbody>
         </table>
     </div>
 </div>
+<br /><br />
+
+<h2 asp-condition="@Model.IsPostBack" class="heading-medium no-bottom-margin">Search results for [@Model.LastSearchTerm]</h2>
 
-<br/><br/>
+<span asp-condition="@Model.IsPostBack && Model.SearchResults.Any() == false">No results</span>
 
+<div asp-condition="@Model.SearchResults.Any()" class="grid-row">
+    <table>
+        <tbody>
+        <tr>
+            <th class="bold">Employer</th>
+            <th class="bold">Vacancy Title</th>
+            <th class="bold">Reference Number</th>
+            <th class="bold">Submitted Date</th>
+            <th class="bold">Closing Date</th>
+            <th class="bold"></th>
+        </tr>
+        </tbody>
+        @foreach (var vacancy in Model.SearchResults)
+        {
+            <tr>
+                <td>@vacancy.EmployerName</td>
+                <td>@vacancy.VacancyTitle</td>
+                <td>@vacancy.VacancyReference</td>
+                <td>@vacancy.SubmittedDate.AsGdsDateTime()</td>
+                <td>@vacancy.ClosingDate.AsGdsDate()</td>
+                <td>
+                    <a asp-condition="@vacancy.IsAvailableForReview" asp-route="@RouteNames.Vacancy_Review_Get" asp-route-reviewId="@vacancy.ReviewId">Review</a>
+                    <span asp-condition="@vacancy.IsAvailableForReview == false">@vacancy.AssignmentInfo</span>
```

Comment:
We've been adding inverses into VMs for this. E.g. `@Model.IsNotAvailableForReview`